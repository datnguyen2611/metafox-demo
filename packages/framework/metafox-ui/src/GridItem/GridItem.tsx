import { Grid, GridProps } from '@mui/material';
import React from 'react';

export default function GridItem(props: GridProps) {
  return <Grid item {...props} />;
}
