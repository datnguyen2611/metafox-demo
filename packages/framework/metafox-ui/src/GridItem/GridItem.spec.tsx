import { GridProps } from '@mui/material';
import { render } from '@testing-library/react';
import * as React from 'react';
import GridItem from './GridItem';

describe('GridItem', () => {
  const props: GridProps = {
    xs: 12,
    children: <div>1</div>
  };

  it('render have children', async () => {
    const { container } = render(<GridItem {...props} />);

    expect(container).toMatchSnapshot();
  });
});
