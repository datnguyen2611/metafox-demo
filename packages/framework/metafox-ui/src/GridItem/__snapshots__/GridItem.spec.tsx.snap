// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridItem render have children 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-13i4rnv-MuiGrid-root"
  >
    <div>
      1
    </div>
  </div>
</div>
`;
